bitcoin/src/psbt/mod.rs

```diff
@@ -857,15 +858,15 @@ mod display_from_str {
 
     impl Display for Psbt {
         fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
-            write!(f, "{}", Base64Display::with_config(&self.serialize(), base64::STANDARD))
+            write!(f, "{}", Base64Display::new(&self.serialize(), &BASE64_STANDARD))
         }
     }
 
     impl FromStr for Psbt {
         type Err = PsbtParseError;
 
         fn from_str(s: &str) -> Result<Self, Self::Err> {
-            let data = base64::decode(s).map_err(PsbtParseError::Base64Encoding)?;
+            let data = BASE64_STANDARD.decode(s).map_err(PsbtParseError::Base64Encoding)?;
             Psbt::deserialize(&data).map_err(PsbtParseError::PsbtEncoding)
```

Comment:
I thought `data` allocation could be avoided, and in fact, it should be possible with `DecoderReader` like shown in the snippet. However, after https://github.com/rust-bitcoin/rust-bitcoin/pull/988 we can't deserialize a PSBT via `Read` trait but only by providing a slice.

```
let mut decoder = base64::read::DecoderReader::new(s.as_bytes(), &BASE64_STANDARD);
Psbt::consensus_decode(&decoder).map_err(PsbtParseError::PsbtEncoding)
```

Reply:
That is quite unfortunate...

I will resolve this due to being out of scope.